src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -0,0 +1,512 @@
+        )}
+        {deliveryMethod === 'sftp' && (
+          <React.Fragment>
+            <div className="row">
+              <div className="col">
+                <ValidationFormGroup
+                  for="sftpHostname"
+                  helpText="The host to deliver the report too"
+                  invalidMessage="Required. Hostname cannot be blank"
+                  invalid={invalidFields.sftpHostname}
+                >
+                  <label htmlFor="sftpHostname">SFTP Hostname</label>
+                  <Input
+                    type="text"
+                    id="sftpHostname"
+                    name="sftpHostname"
+                    defaultValue={config ? config.sftpHostname : undefined}
+                    onBlur={e => this.setState({
+                      invalidFields: {
+                        ...invalidFields,
+                        sftpHostname: !e.target.value.length,
+                      },
+                    })}
+                  />
+                </ValidationFormGroup>
+              </div>
+              <div className="col col-2">
+                <ValidationFormGroup
+                  for="sftpPort"
+                  helpText="The port the sftp host connects too"
+                  invalid={invalidFields.sftpPort}
+                  invalidMessage="Required. Must be a valid port"
+                >
+                  <label htmlFor="sftpPort">SFTP Port</label>
+                  <Input
+                    type="number"
+                    id="sftpPort"
+                    name="sftpPort"
+                    defaultValue={config ? config.sftpPort : 22}
+                    onBlur={e => this.setState({
+                      invalidFields: {
+                        ...invalidFields,
+                        sftpPort: !e.target.value,
+                      },
+                    })}
+                  />
+                </ValidationFormGroup>
+              </div>
+            </div>
+            <div className="row">
+              <div className="col">
+                <ValidationFormGroup
+                  for="sftpUsername"
+                  helpText="the username to securely access the host"
+                  invalidMessage="Required. Username cannot be blank"
+                  invalid={invalidFields.sftpUsername}
+                >
+                  <label htmlFor="sftpUsername">SFTP Username</label>
+                  <Input
+                    type="text"
+                    id="sftpUsername"
+                    name="sftpUsername"
+                    defaultValue={config ? config.sftpUsername : undefined}
+                    onBlur={e => this.setState({
+                      invalidFields: {
+                        ...invalidFields,
+                        sftpUsername: !e.target.value,
+                      },
+                    })}
+                  />
+                </ValidationFormGroup>
+                {/* TODO: check if the password can be blank */}
```

Comment:
Is this still meant as a TODO? Or, should this validation be incorporated into this PR as well?

Reply:
This is being changed now (We got feedback from ECS), so the fields will be changing soon to add a password to both email and sftp methods. Once that is finished, I will remove the TODO